src/main/java/org/gridsuite/study/server/NetworkModificationService.java

```diff
@@ -12,7 +12,7 @@
 import lombok.NonNull;
 import org.apache.commons.lang3.StringUtils;
 import org.gridsuite.study.server.dto.BuildInfos;
-import org.gridsuite.study.server.dto.Receiver;
+import org.gridsuite.study.server.dto.SAReceiver;
```

Comment:
Maybe rename SAReceiver into NodeReceiver, because this receiver stores a node's uuid, and it is used for security analysis and also for the node building process ...

Reply:
agreed, fixed